addon/styles/addon.scss

```diff
@@ -16,6 +16,7 @@ $padding: 0 19px 0 6px;
 .frost-text {
   display: flex;
   align-items: center;
+  width: 173px;
```

Comment:
Please consult with Philippe Savignac on this - at a minimum we should be using units of 5px, but please confirm what the visual designers want the default size to be